Developers/Writing_Linter_Bears.rst

```diff
@@ -387,5 +387,5 @@ Congratulations!
 Where to Find More...
 ---------------------
 
-If you need more information about the ``@linter`` decorator, refer to the API
-documentation.
+If you need more information about the ``@linter`` decorator, refer to the `API
+documentation. <http://docs.coala.io/en/stable/Users/Tutorials/Linter_Bears.html>`_
```

Comment:
thats not literal API documentation, so this doesnt fix the issue. we currently have no api documentation (we had, and we will, but right now we dont), so a good workaround would be to change the phrase to: "If you need more information about the ``@linter`` decorator, refer to the `writing linter bears tutorial. <link>`_"